Pilot/pilotCommands.py

```diff
@@ -861,13 +861,21 @@ def __setInProcessOpts(self):
       localUser = 'Unknown'
     self.log.info('User Name  = %s' % localUser)
     self.log.info('User Id    = %s' % localUid)
-    self.inProcessOpts = ['-s /Resources/Computing/CEDefaults']
-    self.inProcessOpts.append('-o WorkingDirectory=%s' % self.pp.workingDir)
-    self.inProcessOpts.append('-o /LocalSite/CPUTime=%s' % (int(self.pp.jobCPUReq)))
-    self.jobAgentOpts = ['-o MaxCycles=%s' % self.pp.maxCycles,
-                         '-o PollingTime=%s' % self.pp.pollingTime,
-                         '-o StopOnApplicationFailure=%s' % self.pp.stopOnApplicationFailure,
-                         '-o StopAfterFailedMatches=%s' % self.pp.stopAfterFailedMatches]
+    self.innerCEOpts = ['-s /Resources/Computing/CEDefaults']
+    self.innerCEOpts.append('-o WorkingDirectory=%s' % self.pp.workingDir)
+    self.innerCEOpts.append('-o /LocalSite/CPUTime=%s' % (int(self.pp.jobCPUReq)))
+    if self.pp.ceType == 'Pool':
+      self.jobAgentOpts = ['-o MaxCycles=%s' % max(self.pp.pilotProcessors, self.pp.maxCycles),
+                           '-o PollingTime=%s' % min(5, self.pp.maxCycles),
+                           '-o StopOnApplicationFailure=%s' % True and self.pp.stopOnApplicationFailure,
```

Comment:
Why `True and self.pp.stopOnApplicationFailure`?
Isn't it the same thing to just have `self.pp.stopOnApplicationFailure`?

Reply:
Of course yes, thanks! But I think that for PoolCE we should just put `False`...?

Reply:
Yes I think so